docs/source/_src_user_guide/state_propagation/environment_setup/architecture/environment_during_propagation.rst

```diff
@@ -0,0 +1,131 @@
+.. _environment_during_propagation:
+
+===================================================
+Interacting with the environment during propagation
+===================================================
```

Comment:
Top-level comment: I really liked this page overall, and it is extremely useful that you specified how the variables that a user would retrieve are defined by default (for example state w.r.t. global frame origin, etc.). It is something that I myself struggled a lot in the past so very nice to see it clearly documented here :)

Reply:
Thanks! I indeed hope that this page will help with the more advanced use of Tudat